site/content/resources/support.md

```diff
@@ -5,20 +5,51 @@ layout: page
 
 This document describes which versions of Contour are supported by the Contour team.
 
-## Stable release
+## Supported releases
 
-Only the latest stable release is supported.
+Contour is changing both to quarterly releases and three supported releases.
```

Comment:
```suggestion
Contour is releases quarterly and supports three versions starting with Contour v1.20.
```

Reply:
maybe remove `is` in the suggestion as well